pkg/cluster/k8s.go

```diff
@@ -60,27 +60,31 @@ func (k *K8sProvisioner) setUpTiDBCluster(recommend *tidb.TiDBClusterRecommendat
 	if err := k.TestCli.CreateNamespace(recommend.NS); err != nil {
 		return nil, nil, errors.New("failed to create namespace " + recommend.NS)
 	}
-	err = k.TestCli.TiDB.ApplyTiDBCluster(recommend.TidbCluster)
-	if err != nil {
-		return nodes, clientNodes, err
-	}
-
-	err = k.TestCli.TiDB.WaitTiDBClusterReady(recommend.TidbCluster, fixture.Context.WaitClusterReadyDuration)
-	if err != nil {
-		return nodes, clientNodes, err
-	}
 
-	err = k.TestCli.TiDB.ApplyTiDBService(recommend.Service)
-	if err != nil {
-		return nodes, clientNodes, err
+	if cfg, err := k.TestCli.TiDB.GetTiDBCluster(recommend.NS, recommend.NS); err == nil {
```

Comment:
The variable with name "cfg" looks weird to me.

Reply:
Maybe tc is better?